scipy/integrate/_quadpack_py.py

```diff
@@ -700,6 +700,16 @@ def dblquad(func, a, b, gfun, hfun, args=(), epsabs=1.49e-8, epsrel=1.49e-8):
     >>> integrate.dblquad(f, 0, 1, lambda x: x, lambda x: 2-x, args=(3,))
         (0.9999999999999999, 1.6653345369377348e-14)
 
+    Compute the two-dimensional Gaussian Integral, which is the integral of the
+    Gaussian function :math:`f(x,y) = e^{-(x^{2} + y^{2})}`, over
+    :math:`(-\\infty,+\\infty)`. That is, compute the integral
+    :math:`\\iint^{+\\infty}_{-\\infty} e^{-(x^{2} + y^{2})} \\,dy\\,dx`:
+
+    >>> import numpy as np
```

Comment:
Please move the import at the very beginning of the example next to the import to integrate itself.

Same bellow.

Reply:
Ah! I think I see what you mean now, apologies for my misunderstanding. I will change that now!

Reply:
No worries! This is correct, the other example would need the same change.

Reply:
Imagine my face when I realised I had forgotten to change it for the `tplquad` example as well 😅 That change should be pushed now!

Reply:
haha no problem. All good now, thank you! Let's wait for @Kai-Striega now.